app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/prompts/DefaultBrowserPromptsExperimentImpl.kt

```diff
@@ -123,14 +123,23 @@ class DefaultBrowserPromptsExperimentImpl @Inject constructor(
         initialValue = false,
     )
 
+    /**
+     * Model used to parse remote config setting. All values are integer strings, for example "1" or "20".
+     */
     @VisibleForTesting
-    data class FeatureSettings(
+    data class FeatureSettingsConfigModel(
+        val activeDaysUntilStage1: String,
+        val activeDaysUntilStage2: String,
+        val activeDaysUntilStop: String,
```

Comment:
Why do we use strings when this are `Int` types in reality?

Reply:
I am not sure it's true, look at `ampLinks.settings`, it has `deepExtractionTimeout` which is an Int and other params that are eg. boolean values.

I am not sure if "just an experiment" is a good argument for not doing things the right way tbf.

Reply:
`ampLinks.settings` is top-level feature `settings` object, the problem we hit here is with the nested feature `settings` object - the schema unit tests were asking for strictly string values.

Looks like we'd need to tweak the way we define the `SubFeature` to allow the `SettingsType` to be at least `Record<string, string | number>` (if not just `any`), right now this seems to [default to `Record<string, string>`](https://github.com/duckduckgo/privacy-configuration/blob/10cec00018890d882b42877b65a76d187d888234/schema/feature.ts#L20) which would explain the required `string` value by unit tests. I can try a couple of things to make it work.

Reply:
I think it's fine, leave it as is. Thanks for looking into it